modules/tinymce/src/plugins/advlist/test/ts/browser/AdvlistOptionsAndToolbarTest.ts

```diff
@@ -77,158 +72,233 @@ describe('browser.tinymce.plugins.advlist.AdvlistOptionsAndToolbarTest', () => {
     ));
   };
 
-  Arr.each([
-    {
-      label: 'TBA: Test default advlist_number_styles option behaviour',
-      options: {},
-      splitBtns: { number: true, bullet: true },
-      type: 'number',
-      expectedContent: '<ol>\n<li>a</li>\n</ol>'
-    },
-    {
-      label: 'TBA: Test default advlist_bullet_styles option behaviour',
-      options: {},
-      splitBtns: { number: true, bullet: true },
-      type: 'bullet',
-      expectedContent: '<ul>\n<li>a</li>\n</ul>'
-    },
-    {
-      label: 'TINY-8721: Test not including default in advlist_number_styles option',
-      options: {
-        advlist_number_styles: 'lower-alpha,lower-greek'
+  const pAssertButtonToggledState = (name: string, state: boolean) =>
+    Waiter.pTryUntil('Wait for toolbar button state', () => {
+      const button = UiFinder.findIn(SugarBody.body(), `div.tox-split-button[aria-label="${name}"]`).getOrDie();
+      return Assertions.assertStructure('', ApproxStructure.build((s, _, __) => s.element('div', {
+        children: [
+          s.element('span', {
+            exactClasses: [ 'tox-tbtn', ...(state ? [ 'tox-tbtn--enabled' ] : [] ) ]
```

Comment:
You can also use `arr.has` and `arr.not`, but I don't want to stand in the way of you using your new API :)